src/AllocatorBuffer.sol

```diff
@@ -84,15 +69,13 @@ contract AllocatorBuffer is IAllocatorConduit {
         emit Approve(asset, spender, amount);
     }
 
-    function deposit(bytes32, address asset, uint256 amount) external {
+    function deposit(address asset, uint256 amount) external {
```

Comment:
Do we still need the `deposit` and `withdraw` functions here?
I'm not sure as it seems the conduits, swapper, and depositor don't use them.
Maybe just for future support if needed?

Reply:
Removed.